src/main/java/seedu/address/model/travelplan/exceptions/DuplicateTravelPlanException.java

```diff
@@ -1,11 +1,13 @@
 package seedu.address.model.travelplan.exceptions;
 
+import seedu.address.commons.exceptions.DuplicateException;
+
 /**
  * Signals that the operation will result in duplicate TravelPlans (TravelPlans are considered duplicates if they have
  * the same
  * identity).
  */
-public class DuplicateTravelPlanException extends RuntimeException {
+public class DuplicateTravelPlanException extends DuplicateException {
     public DuplicateTravelPlanException() {
         super("Operation would result in duplicate persons");
```

Comment:
duplicate friends?

Reply:
updated to duplicate travel plans! :) thanks!